app/components/Table/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+
+export default class Table extends Component {
+  render() {
+    const { data } = this.props
+
+    let columns = Object.keys(data[0])
+    return (
+      <table>
```

Comment:
i wonder if we can reuse this but with some class placeholder here for semantic-ui like 

```html
<table className={...
```